src/services.js

```diff
@@ -136,16 +156,21 @@ const checkStatus = async function (tilesetId) {
     } else if (response.body.status === "success") {
       console.log(`Complete: opening https://studio.mapbox.com/tilesets/${response.body.id}/`);
       open(`https://studio.mapbox.com/tilesets/${response.body.id}/`, { url: true });
+      // Nice to also have tileStats printed, but this could be swapped for checkJobWarnings
```

Comment:
I think I prefer `checkJobWarnings` here, rather than have the tile stats. Can we move it up to the top of this if block?